src/JackTrip.cpp

```diff
@@ -211,6 +212,10 @@ void JackTrip::setupAudio(
     }
 
     mAudioInterface->setLoopBack(mLoopBack);
+    if (mAudioTesterP) { // if we're a hub server, this will be a nullptr - MAJOR REFACTOR NEEDED, in my opinion
+      mAudioTesterP->setSampleRate(mSampleRate);
+    }
+    mAudioInterface->setAudioTesterP(mAudioTesterP);
```

Comment:
should this be inside the if{} block? what is the impact of setting this to a nullptr when in hub server mode?

Reply:
AudioInterface needs the null pointer so it can check it before using it.  This has been tested in hub server mode.

Reply:
Furthermore, -x is not allowed in server modes (jacktrip exits when -S or -s is used with -x).